diff -urN aaa/atomix-1.0/atomix.desktop.in atomix-1.0/atomix.desktop.in
--- aaa/atomix-1.0/atomix.desktop.in	2002-12-05 20:38:39.000000000 +0100
+++ atomix-1.0/atomix.desktop.in	2003-12-12 16:33:39.000000000 +0100
@@ -5,5 +5,5 @@
 Icon=
 Terminal=false
 Type=Application
-Categories=Application;Game;MindGame
-
+Categories=GTK;GNOME;Game;LogicGame;
+Encoding=UTF-8
--- atomix-1.0.1/configure.in~	Wed Nov 10 12:37:42 2004
+++ atomix-1.0.1/configure.in	Wed Nov 10 12:37:42 2004
@@ -40,7 +40,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 
 dnl Add the languages which your application supports here.
-ALL_LINGUAS="am az be ca cs da de el es et fa fr gl he hi hu it ja kn lv ms nl no pl pt pt_BR ru sk sl sq sr sr@Latn sv ta tr uk vi wa zh_CN zh_TW ko"
+ALL_LINGUAS="am az be ca cs da de el es et fa fr gl he hi hu it ja kn lv ms nb nl pl pt pt_BR ru sk sl sq sr sr@Latn sv ta tr uk vi wa zh_CN zh_TW ko"
 AM_GLIB_GNU_GETTEXT
 
 EXTRA_WARNING_CFLAGS=" $warning_flags"
